i18n/messages-en.xtb

```diff
@@ -180,6 +180,7 @@
   <translation id="6556203351766206955" key="MSG_DEPLOYMENTDETAIL_DEPLOYMENTINFO_2" desc="Label \'Namespace\' for the deployment namespace on the deployment details page.">Namespace</translation>
   <translation id="7408059421363536605" key="MSG_DEPLOYMENTDETAIL_DEPLOYMENTINFO_3" desc="Label \'Labels\' for the deployment\'s labels list on the deployment details page.">Labels</translation>
   <translation id="1093338324719768337" key="MSG_DEPLOYMENTDETAIL_DEPLOYMENTINFO_4" desc="Label \'Label selector\' for the deployment\'s labels list on the deployment details page.">Label selector</translation>
+  <translation id="4730072908657325030" key="MSG_DEPLOYMENTDETAIL_DEPLOYMENTINFO_4" desc="Label \'Selector\' for the deployment\'s labels list on the deployment details page.">Selector</translation>
```

Comment:
Let me just make sure if these changes are intended - because they don't seem related to TPRs.
(If they are indeed unrelated but it's convenient for you to have them in this PR, then OK).